Resources/Prototypes/_NF/Loadouts/role_loadouts.yml

```
@@ -204,6 +204,27 @@
   - ContractorBureaucracy
   - NFSpeciesSpecific
 
+- type: roleLoadout
+  id: JobDirectorofCare
+  groups:
+  - DocHead
+  - DocNeck
+  - DocJumpsuit
+  - MercenaryGloves
+  - MercenaryBackpack  
+  - ContractorShoes
+  - ContractorFace
+  - ContractorGlasses
+  - DocPDA
+  - ContractorBelt
+  - DocEars
+  - ContractorBoxSurvival
+  - ContractorCartridge
+  - ContractorImplanter
+  - ContractorFun
+  - ContractorTrinkets
+  - ContractorBureaucracy
```

```suggestion
  - ContractorBureaucracy
  - NFSpeciesSpecific
```
If you'd prefer the vox to have a medical mask by default, I can get you an NFMedicalSpeciesSpecific definition